pkg/config/xds/xds.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/golang/protobuf/proto"
 
 	networking "istio.io/api/networking/v1alpha3"
+	// TODO(gargnupur): Remove  this after https://github.com/envoyproxy/envoy/issues/9640
+	// is fixed. Pilot needs descriptors for metadata  exchange config so that it can be
+	// applied via EnvoyFilter.
+	_ "github.com/istio/proxy/src/envoy/tcp/metadata_exchange/config"
```

Comment:
Add this to filter_types.go that was merged yesterday. We are importing all the filters there.

Reply:
Thanks.. Fixed